crates/control/src/motion/motion_selector.rs

```diff
@@ -103,15 +104,17 @@ fn motion_type_from_command(
 fn transition_motion(
     from: MotionType,
     to: MotionType,
-    is_safe_to_exit: bool,
+    motion_finished: bool,
```

Comment:
What was your reasoning for the renaming?

Reply:
The meaning of the variable changed. In the standUpBack, standUpFront motions this will now also be set when the motion is aborted. This allows for a repeated dispatch to the initial_positions of the standup motion.

Reply:
But shouldn't the motion also be safe to exit on abort? The motion is not finished when it is aborted. Therefore I'm wondering whether the renaming is better. :thinking: